config/tileset/aerial.json

```diff
@@ -569,7 +569,7 @@
       "2193": "s3://linz-basemaps/2193/waikato_urban_2014_0-10m_RGBA/01F6P1ZQZ4X5GHHX544CJJ3NNH",
       "3857": "s3://linz-basemaps/3857/waikato_urban_2014_0-10m_RGBA/01ED83NZS98MYTMWM4D1EVZ07T",
       "name": "waikato_urban_2014_0-10m_RGBA",
-      "minZoom": 14,
+      "minZoom": 32,
```

Comment:
The layers prioritising in `aerial.json` are depends on the order of them. The later in the list has a higher priority. So to deprioritise layers, we usually move up in this list instead of changing zoom level. Change zoom level to 32 mean stop showing this layer on the map.

Reply:
Ah, I meant stop showing. This one will be removed once we have an 'all configuration'.